CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.1.0 - 2024/07/09
```

Comment:
Thanks so much for updating the CHANGELOG 😄 . Could you also update the version in `nextflow.config` to match.

Reply:
Thanks :)